client/starwhale/consts/env.py

```diff
@@ -8,6 +8,7 @@ class SWEnv:
     status_dir = "SW_TASK_STATUS_DIR"
     log_dir = "SW_TASK_LOG_DIR"
     dataset_uri = "SW_DATASET_URI"
+    finetune_validation_dataset_uri = "SW_FINETUNE_VALIDATION_DATASET_URI"
```

Comment:
The server env named as 'SW_VALIDATION_DATASET_URI', so, keep consistent with server? cc @anda-ren

Reply:
It's fine, I could modify it in the server side

Reply:
ok